src/components/Landing/MoreWays/index.js

```diff
@@ -0,0 +1,54 @@
+import { Subtitle } from '@leafygreen-ui/typography';
+import styled from '@emotion/styled';
+import { cx, css } from '@leafygreen-ui/emotion';
+
+import Video from '../../Video';
+import ComponentFactory from '../../ComponentFactory';
+import { theme } from '../../../theme/docsTheme';
+
+const StyledContainer = styled('div')`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  grid-column-start: 2;
+  grid-column-end: 14;
+  max-width: 1440px;
+  gap: 2rem;
```

Comment:
```suggestion
  column-gap: 2rem;
```

![image](https://github.com/mongodb/snooty/assets/13054820/e68ec897-6d1e-411b-9873-bc43b5bd6f32)


seeing a minor margin on mobile view